projects/NodeBB/services/nodebb/examples/postgresql.nix

```diff
@@ -22,8 +22,9 @@
 
   # demo-vm
   networking.firewall.allowedTCPPorts = [ 4567 ];
```

Comment:
We can make an `openFirewall` option for this, or at least get the service port from `settings`.